src/macaroni/app/App.java

```diff
@@ -24,23 +20,30 @@ public void run() {
         running = true;
         window.open();
 
+        long fps = 60;
```

Comment:
Why wasn't the original `FPS` good?

Reply:
Unnecessary class attribute, only used here